lmfit/models.py

```diff
@@ -735,6 +735,60 @@ def guess(self, data, x, negative=False, **kwargs):
     guess.__doc__ = COMMON_GUESS_DOC
 
 
+class Pearson4Model(Model):
+    r"""A model based on a Pearson IV distribution.
+
+    The model has five parameters: `amplitude` (:math:`A`), `center`
+    (:math:`\mu`), `sigma` (:math:`\sigma`), `expon` (:math:`m`) and `skew` (:math:`\nu`).
+    In addition, parameters `fwhmapprox`, `height` and `position` are included as
+    constraints to report estimates for the approximate full width at half maximum, the
+    peak height, and the peak position, respectively.
+    The fwhmapprox value has an error of about 20% in the
+    parameter range expon: (0.5, 1000], skew: [-1000, 1000].
+
+    .. math::
+
+        f(x;A,\mu,\sigma,m,\nu)=A \frac{\left|\frac{\Gamma(m+i\tfrac{\nu}{2})}{\Gamma(m)}\right|^2}{\sigma\beta(m-\tfrac{1}{2},\tfrac{1}{2})}\left[1+\frac{(x-\mu)^2}{\sigma^2}\right]^{-m}\exp\left(-\nu \arctan\left(\frac{x-\mu}{\sigma}\right)\right)
+
+    where :math:`\beta` is the beta function (see :scipydoc:`special.beta`).
+    The :meth:`guess` function always gives a starting value of 1.5 for `expon`,
+    and 0 for `skew`. In addition, parameters `fwhmapprox`, `height` and `position` are included as
```

Comment:
why is the parameter called `fwhmapprox`; this is in other lineshapes sometimes also an approximation - correct? I'd prefer to keep the names consistent and just call it here `fwhm` as well.